esx_service/vmdk-opsd

```diff
@@ -150,7 +184,7 @@ check_running_status () {
    fi
 
    echo "${OPSD_TAG} is not running"
-   return 6
+   return 5
```

Comment:
why 5 and not 6 ? I think these numbers should be constants, but if you use literals - why change value ?

Reply:
@lipingxue asked the same question. Please see my comments above, and let me know your opinion.